autogen/agentchat/contrib/capabilities/generate_images.py

```diff
@@ -108,6 +114,12 @@ def cache_key(self, prompt: str) -> str:
         keys = (prompt, self._model, self._resolution, self._quality, self._num_images)
         return ",".join([str(k) for k in keys])
 
+    def _dalle_client_factory(self, dalle_config: Dict) -> Union[OpenAI, AzureOpenAI]:
+        if dalle_config.get("api_type") == "azure":
+            return AzureOpenAI(api_key=dalle_config["api_key"])
+        else:
+            return OpenAI(api_key=dalle_config["api_key"])
```

Comment:
For OpenAI, I'm not sure if more keys are possible. Why do we pass the `api_key` only here?